README.md

```diff
@@ -42,22 +41,42 @@ If you wish to set up a conda environment for using `sidora.core`, instructions
     remotes::install_github("sidora-tools/sidora.core")                             
     ``` 
 
+## Quickstart guide
 
+To use this package you have to follow these steps:
 
-## Quickstart guide
+1. Install the sidora.core package in R (see above)
+2. Create a `.credentials` file\*
+3. Connect to the institutes subnet via VPN (see the instructions in kbase)
+4. Establish an ssh tunnel to the pandora database server with
 
-Load the package and establish a database connection to Pandora. To do so you need the right `.credentials` file. Contact Stephan Schiffels, James Fellows Yates, or Clemens Schmid to obtain it. You also have to be in the institute's subnet.
+    ```bash
+    ssh -L 10001:pandora.eva.mpg.de:3306 <your username>@daghead1
+    ```
```

Comment:
```suggestion
    ```

    > You must make a new tunnel each time you want to connect (e.g. after you log out or reboot your machine)
```